### PR TITLE
fix/async-doc

### PR DESCRIPTION
**Pull request by Eomm**
Related to #35

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +42,10 @@ fastify
     })
   })
 ```
+
+This plugin support `callback` and `Promise` returned by the functions. Note that an `async` function
+doesn't have to use the `done` parameter.
```
Comment (mcollina) [inline README.md]: Can you include the docs for callbacks as well?

Comment (Eomm) [reply README.md]: I should have added a better description

Comment (mcollina) [reply README.md]: I mean, having both example is better.

Comment (Eomm) [reply README.md]: Done 👍

**Pull request merged**